app/src/main/java/com/github/se/polyfit/viewmodel/post/ViewPostViewModel.kt

```diff
@@ -19,17 +27,43 @@ import kotlinx.coroutines.flow.StateFlow
 @HiltViewModel
 class ViewPostViewModel
 @Inject
-constructor(private val postFirebaseRepository: PostFirebaseRepository) : ViewModel() {
+constructor(
+    private val postFirebaseRepository: PostFirebaseRepository,
+    private val postLocalRepository: PostLocationModel
+) : ViewModel() {
   private val _posts: MutableStateFlow<List<Post>> = MutableStateFlow(mutableListOf())
+  val posts: StateFlow<List<Post>> = _posts
```

Comment:
If the other variable were changed to LiveData, it might be a good idea, for the sake of consistency, to also make this one LiveData.

Reply:
I tried it doesn't work